src/bin/new.rs

```diff
@@ -22,29 +25,53 @@ Usage:
     cargo new -h | --help
 
 Options:
-    -h, --help          Print this message
-    --vcs VCS           Initialize a new repository for the given version
-                        control system (git or hg) or do not initialize any version
-                        control at all (none) overriding a global configuration.
-    --bin               Use a binary instead of a library template
-    --name NAME         Set the resulting package name
-    -v, --verbose       Use verbose output
-    -q, --quiet         No output printed to stdout
-    --color WHEN        Coloring: auto, always, never
-";
+    -h, --help              Print this message
+    --vcs VCS               Initialize a new repository for the given version
+                            control system (git or hg) or do not initialize any version
+                            control at all (none) overriding a global configuration.
+    --bin                   Use a binary instead of a library template
+    --name NAME             Set the resulting package name
+    -v, --verbose           Use verbose output
+    -q, --quiet             No output printed to stdout
+    --color WHEN            Coloring: auto, always, never
+    --license LICENSES      License(s) to add to a project
+                            Multiple licenses should be separated by a '/' character
```

Comment:
Right now crates.io actually also supports some more flavorful strings of licenses, and I'd kinda prefer to avoid parsing license strings everywhere and instead just keep it on crates.io for now. Perhaps this help documentation could just be "add a license to a project" and if it's understood we'll write some files out but if not a warning will be emitted that the license files themselves don't exist?